CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # bedrock-meter-http ChangeLog
 
+## 3.0.0 - 2021-09-xx
+- **BREAKING**: A capability invocation via http-sigs is required to create
+  meters. Access is controlled via the `config.createMeterAllowList`
```

Comment:
```suggestion
  meters. Access is controlled via the `config.meterCreationAllowList`
```

Reply:
I'd use this bikeshed color because as a noun, it can't be confused with the verby concept of "setting this config var will cause the 'meter allow list' to be created".

Reply:
No worries, I wanted to save a few characters.